ha/k8s_setup/ha_setup.py

```diff
@@ -187,11 +188,14 @@ def process(self):
         Process config command.
         """
         try:
-            consul_endpoint = Conf.get(self._index, f'cortx{_DELIM}external{_DELIM}consul{_DELIM}endpoints[0]')
-            if not consul_endpoint:
+            consul_endpoints = Conf.get(self._index, f'cortx{_DELIM}external{_DELIM}consul{_DELIM}endpoints')
+            # search for supported consul endpoint url from list of configured consule endpoints
+            consul_endpoint = list(filter(lambda x: urlparse(x).scheme == const.consule_scheme, consul_endpoints))
```

Comment:
Can you please add snippet from cluster.conf in PR comments; where consulu endpoints are listed 
This will help in understanding the logic that is used

Reply:
can you please change the name of this variable. May be you can keep it as required_consul_endpoint_list. You can specify the type also there.

Reply:
done

Reply:
> 

Both comments are done

Reply:
Thanks!